### ISSUE TITLE
Seach for 3rd party libraries

### ISSUE THREAD

**Issue by doraess** [amperka/ino#50]
Is there any way to declare where are 3rd party library are installed?
The usual path for those in Arduino IDE ("sketckbook/libraries") are not found. 
It is not useful to move all needed libraries into lib folder for each project, when those are already installed ad recognized by Arduino IDE.

**Comment by nkrkv** [comment]
Yep. This feature is a must. So it will be added in the next version.

**Comment by jblb** [comment]
I agree with this this feature is a must be !

**Comment by mchughj** [comment]
Hi Nailxx.  Are you still planning on implementing this feature?  Mind if I have a go at it if you don't have it already in flight?  Thanks.  Jason

**Comment by nkrkv** [comment]
Yep, I'll try to do it this week. Unfortunately there is no so much spare time.

**Comment by mchughj** [comment]
I totally understand the lack of free time.  I went ahead and made this change:

  https://github.com/mchughj/ino/commit/e55df229cb4ab28300f15d795d91613c25ae496a

I'm new to a lot of things in the github world so I don't quite know how this works.  If this was what you were thinking of doing but want some changes then let me know and I would be happy to make whatever modifications you feel are correct.  If this isn't what you wanted at all then just let me know.  :)

**Comment by nkrkv** [comment]
Thanks! It looks like what we need.

Will look more precisely and do the merge soon.

**Comment by mchughj** [comment]
Sounds great.  Please let me know if you think I missed anything and I would be happy to make additional modifications.

**Comment by tyilo** [comment]
The default directory for OS X installations is `~/Documents/Arduino/libraries`, could you also include that directory?

**Comment by mchughj** [comment]
Victor - what do you think?
